virttest/utils_test/libvirt.py

```diff
@@ -1089,10 +1091,15 @@ def thread_func_migration(self, vm, desturi, options=None):
             if options is None:
                 options = "--live --timeout=60"
             stime = int(time.time())
-            vm.migrate(desturi, option=options, ignore_status=False,
-                       debug=True)
+            self.ret = vm.migrate(desturi, option=options, ignore_status=ignore_status,
+                                  debug=True)
             etime = int(time.time())
             self.mig_time[vm.name] = etime - stime
+            if ignore_status is True and self.ret is not None:
```

Comment:
Is this check necessary here? Since you ignored the failure and what' following is in do_migration function which will release lock if the thread is alive.